
float64[] goal
float64 duration
bool gravityIdentification
---
bool success